content/en/docs/Responses/AlbumID3WithSongs.md

```diff
@@ -161,11 +195,44 @@ description: >
 | `genre` | `string` | No |     | The album genre|
 | `played` | `string` | No | **Yes**    | Date the album was last played. [ISO 8601]|
 | `userRating` | `int` | No | **Yes**    | The user rating of the album. [1-5]|
+| `albumLabel` | `string` | No |  **Yes**   | The label producing the album. |
+| `musicBrainzId` | `string` | No |  **Yes**   | The album MusicBrainzID. |
+| `genres` | Array of `string` | No | **Yes**    | The list of all genres of the album. |
+| `artists` | Array of [`ArtistID3`](../artistid3) | No | **Yes**    | The list of all album artists of the album. |
+| `displayArtist` | `string` | No |  **Yes**   | The single value display artist. |
+| `releaseType` | `string` | No | **Yes**    | The type of release. If the server handle subtypes then the field should return them type and subtypes separated with " / ".|
+| `moods` | Array of `string` | No | **Yes**    | The list of all moods of the album. |
+| `sortName` | `string` | No |  **Yes**   | The album sort name. |
+| `originalReleaseDate` | `string` | No |   **Yes**   | Date the album was originaly released. [ISO 8601]|
+| `isCompilation` | `boolean` | No |  **Yes**    | True if the album is a compilation.|
+| `discTitles` | Array of [`DiscTitle`](../disctitle) | No | **Yes**    | The list of all disc titles of the album. |
 | `song` | Array of [`Child`](../child) | No |     | The list of songs |
 
 {{< alert color="warning" title="OpenSubsonic" >}}
 New fields are added:
 
 - `played`
 - `userRating`
+- `albumLabel`
+- `musicBrainzId`
+- `genres`
+- `artists`
+- `displayArtist`
+- `releaseType`
+- `moods`
+- `sortName`
+- `originalReleaseDate`
+- `isCompilation`
```

Comment:
how do we specifically define complication here? can it be deduced from `len(artists) > 1`, or is it based on file metadata, where user may have an album which has multiple artists but still not a compilation?

Reply:
I think it's based on file metadata. There is a COMPILATION ID3 tag that Navidrome and iTunes and others use, to identify an album, say like "Top 90s Hits" that has a different artist for each track. (And the album artist is often tagged as Various Artists in this case)

Reply:
Yes there's a couple of tags for that, but I do not want to be more explicit as a server might offer a way for an user to set an album as a compilation manually.